### PR TITLE
GE PMT: Add 'Do Not Charge Card' feature to Payments widget

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom Labels
  - geBodyPaymentNotProcessingPayment
  - geButtonPaymentAlternate
  - geButtonPaymentDoNotCharge
# Deleted Metadata

### CODE REVIEW

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -5617,6 +5625,22 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <protected>true</protected>
+        <shortDescription>Link to toggle and reactivate payment processing fields</shortDescription>
+        <value>Alternatively, charge a card now.</value>
+    </labels>
+    <labels>
+        <fullName>geButtonPaymentDoNotCharge</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link to toggle off payment processing</shortDescription>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels:5641]:
```suggestion
        <shortDescription>Link to toggle payment processing off</shortDescription>
```

**Comment by melissabarber** [comment]
@psadie-sfdo added one minor change to a shortDescription. After that, it's good to go!

**Comment by psadie-sfdo** [comment]
> @psadie-sfdo added one minor change to a shortDescription. After that, it's good to go!

Thank you!

**Comment by psadie-sfdo** [comment]
**lurch: attach W-040601

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HuNNQA0" target="_blank">W-040601</a>

**Pull request merged**